javascript/javascript1/week1/answers.js

```diff
@@ -0,0 +1,83 @@
+/* 
+//************************************************ 
+STEP2 answer
+//************************************************ 
+
+Alejandro Gispert
+https://www.freecodecamp.org/AlejandroGispert
+
+*/
+
+//************************************************ 
+//Age-ify (A future age calculator) 
+//************************************************ 
+
+const yearOfBirth = 1986
+const yearFuture = 2045
+
+const age = yearFuture - yearOfBirth
+
+console.log("You will be " + age +" years old in " + yearFuture)
+
+//************************************************ 
+//Goodboy-Oldboy (A dog age calculator)
+//************************************************ 
+
+
+const dogYearOfBirth = 2010
+const dogYearFuture = 2045
+let dogYear = dogYearFuture - dogYearOfBirth
+const shouldShowResultInDogYears = true
+let years = "human years"
+
+if (shouldShowResultInDogYears) {
+years = "dog years"
+   dogYear *= 5
+}
+console.log("Your dog will be " + dogYear +" "+ years +" old in " + dogYearFuture )
+
+
+
+//************************************************ 
+//Housey pricey (A house price estimator)
+//************************************************ 
+const length = 5   
+const width = 11
+const depth = 8
+
+const Peter = [8,10,10]
+const Julia =[length,width,depth]
+
+const volumeInMetersPeter = Peter[0] * Peter[1] * Peter[2]
+const volumeInMetersJulia = Julia[0] * Julia[1] * Julia[2]
+
+const gardenSizeInM2Peter =  100;
+const gardenSizeInM2Julia =  70;
+
+const correctHousePricePeter = volumeInMetersPeter * 2.5 * 1000 + gardenSizeInM2Peter * 300;
+const correctHousePriceJulia = volumeInMetersJulia * 2.5 * 1000 + gardenSizeInM2Julia * 300;
+
+function numberWithPunctum(num) {
+    return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ".");
+}
```

Comment:
Only if you have the time, but could you look into functions and see how could you improve the entire task? You could save many lines of code, but functions are also something that hasn't been thought yet.

Reply:
done